spring-cloud-skipper-shell-commands/src/main/java/org/springframework/cloud/skipper/shell/command/PackageCommands.java

```diff
@@ -49,10 +65,60 @@ public PackageCommands(SkipperClient skipperClient) {
 		this.skipperClient = skipperClient;
 	}
 
-	@ShellMethod(key = "package list", value = "Get the package metadata")
-	public String packageMetadata(
-			@ShellOption(help = "boolean to set for more detailed package metadata", defaultValue = "false") boolean details) {
-		return skipperClient.getPackageMetadata(details);
+	@ShellMethod(key = "package search", value = "Search for the packages")
+	public Object searchPackage(
+			@ShellOption(help = "wildcard expression to search for the package name", defaultValue = NULL) String name,
```

Comment:
Careful the default is null, which is not tested in the implementation, is that correct?